cookbooks/README.md

```diff
@@ -94,100 +89,90 @@ To:
 </cluster>
 ```
 
-
-
 In case you want to customize your installation using roles, you can declare attributes like this: 
 
 ```json
 {
-    "name": "wazuh_manager",
-    "description": "Wazuh Manager host",
-    "json_class": "Chef::Role",
-    "default_attributes": {
-        "ossec": {
-            "cluster":{
-                "disabled" : "false"
-            }
+  "name": "wazuh_server",
+  "description": "Wazuh Server Role",
+  "json_class": "Chef::Role",
+  "default_attributes": {
+    "ossec": {
+        "cluster":{
+            "disabled" : "false"
         }
-    },
-    "override_attributes": {
+    }
+  },
+  "override_attributes": {
 
-    },
-    "chef_type": "role",
-    "run_list": [
-      "recipe[wazuh_manager::manager]"
-    ],
-    "env_run_lists": {
+  },
+  "chef_type": "role",
+  "run_list": [
+    "recipe[wazuh_manager::default]",
+    "recipe[filebeat::default]"
+  ],
+  "env_run_lists": {
 
-    }
   }
+}
 ```
 
-The same example applies for Wazuh Agent and it's own attributes.
+The same example applies for the rest of cookbooks and it's own attributes.
 
 You can get more info about attributes and how the work on the chef documentation: https://docs.chef.io/attributes.html
 
-
-
 ### Centralized Configuration
 
-You can configure your Wazuh [Centralized Configuration](https://documentation.wazuh.com/3.9/user-manual/reference/centralized-configuration.html#centralized-configuration-process) with Chef.
+You can configure your Wazuh [Centralized Configuration](https://documentation.wazuh.com/4.0/user-manual/reference/centralized-configuration.html#centralized-configuration-process) with Chef.
```

Comment:
You can set up you Wazuh...

Also, please, consider using current instead of fixed version